partiql-logical/src/graph/mod.rs

```diff
@@ -24,6 +25,12 @@ pub enum DirectionFilter {
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 pub struct BindSpec(pub String);
 
+impl BindSpec {
+    pub fn is_anon(&self) -> bool {
```

Comment:
What is `is_anon` used for? Could add some description

Reply:
Will do. Basically, this is an 'anonymous' bind-name. Kind of like the PartiQL spec uses `_1` etc. for synthetic variables, the GPML paper explains pattern expansions via the insertion of anonymous bind variables.